src/sap/fhir/model/r4/lib/FHIROperationOutcome.js

```diff
@@ -0,0 +1,117 @@
+/*!
+ * ${copyright}
+ */
+
+// Provides class sap.fhir.model.r4.lib.FHIROperationOutcome
+sap.ui.define([], function () {
+	"use strict";
+
+	/**
+	 * Constructor for a new FHIROperationOutcome. [FHIR Specification]{@link https://www.hl7.org/fhir/operationoutcome.html}
```

Comment:
I would move this section "[FHIR Specification]{@link https://www.hl7.org/fhir/operationoutcome.html}" to @see [FHIR Specification]{@link https://www.hl7.org/fhir/operationoutcome.html}

Reply:
done